traffic-policy/actions/oauth/examples/oauth-headers.mdx

```diff
@@ -0,0 +1,9 @@
+### Upstream Headers {#upstream-headers}
+
+| Header                                 | Description                                                                 |
```

Comment:
we can remove these i think?

Reply:
removed here: https://github.com/ngrok/ngrok-docs/pull/1073/commits/251a768a122a293f2036d822dc1c2280ea36edd5